lib/rubycritic/generators/html_report.rb

```diff
@@ -3,15 +3,18 @@
 require "rubycritic/generators/html/smells_index"
 require "rubycritic/generators/html/code_index"
 require "rubycritic/generators/html/code_file"
+require "rubycritic/generators/global_rating_calculator"
 
 module Rubycritic
   module Generator
 
     class HtmlReport
+      include GlobalRatingCalculator
       ASSETS_DIR = File.expand_path("../html/assets", __FILE__)
 
       def initialize(analysed_modules)
         @analysed_modules = analysed_modules
+        @gpa = calculate_gpa(analysed_modules)
```

Comment:
I don't think it's a `Report`'s responsibility to know how to calculate the GPA. Make `GlobalRatingCalculator` a class and call this method on it!